### PR TITLE
[Enhancement] Add "Company Logo" label for dropzone in settings

### LINKED ISSUE #483: [Enhancement] Add "Company Logo" label for dropzone in settings

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #483 

image

### CODE REVIEW

File: app/styles/granite/page/settings.less [hunk 1]
```diff
@@ -14,3 +14,12 @@
...
 }
+
+label[for="input__dropzone--company-image"]{
+  display: block;
+  margin: 0em 0em 0.28571429rem 0em;
+  color: #3E464D;
+  font-size: 0.92857143em;
+  font-weight: bold;
+  text-transform: none;
+}
```
Comment (James1x0) [inline app/styles/granite/page/settings.less]: Is this needed, aren't labels styled by default? 🤔

Comment (JavaLavaMT) [reply app/styles/granite/page/settings.less]: I just thought you wanted the label to match the other labels, below is the label without additional styling. If you are ok with this, I will send the fix 

image

Comment (James1x0) [reply app/styles/granite/page/settings.less]: Labels need a surrounding `.ui.form` to be styled. There's no additional styling needed.

**Comment by JavaLavaMT** [comment]
image

**Pull request merged**